tests/Umbraco.Tests.UnitTests/Umbraco.Core/PropertyEditors/SliderPropertyValueEditorTests.cs

```diff
@@ -28,17 +31,31 @@ public class SliderPropertyValueEditorTests
         true,
         new object(),
         new List<string> { "some", "values" },
-        Guid.NewGuid(),
-        new GuidUdi(Constants.UdiEntityType.Document, Guid.NewGuid())
+        // NewGuid, // This is a valid Guid, but not a valid value for Can_Handle_Invalid_Values_From_Editor test.
```

Comment:
Please just remove this rather than commenting it out.

Reply:
Done.